astropy/modeling/functional_models.py

```diff
@@ -33,6 +35,17 @@ class BaseGaussian1D(Fittable1DModel):
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
```

Comment:
Maybe @nden can confirm. I was just adapting from incantations I found from discussions in other issue and PR.